react/src/components/Footer.js

```diff
@@ -1,41 +1,69 @@
+import * as Sentry from '@sentry/react';
+import { Fragment } from 'react';
 import { Component } from 'react';
-import { Link } from 'react-router-dom';
 import './footer.css';
 
 class Footer extends Component {
+  state = {
+    subscribed: false,
+  };
+
   async shouldComponentUpdate() {
     console.log('> Footer shouldComponentUpdate');
   }
 
   render() {
+    const handleSubscribeClick = () => {
+      this.setState({ subscribed: true });
+    };
+
     return (
       <footer id="footer">
         <div>
           <h2 className="h3 sentry-unmask">Sign up for plant tech news</h2>
-          <div class="formContainer">
-            <form>
-              <label htmlFor="email-subscribe" className="sentry-unmask">
-                Email
-              </label>
+          <Sentry.ErrorBoundary
+            onReset={() => {
+              this.setState({ subscribed: false });
+            }}
+            fallback={({ resetError, eventId }) => {
+              return (
+                <Fragment>
+                  <div>An error has occurred. Sentry Event ID: {eventId}</div>
+                  <button className="btn" onClick={resetError}>
+                    Reset Form
+                  </button>
```

Comment:
I made a quick call on copy, but we can change to anything else if it fits better.